### PR TITLE
tutorial INBO citation style

### PR DESCRIPTION
**Pull request by hansvancalster**
Implements the tutorial mentioned in issue #168.

@florisvdh can you render this to html so the contents of the tutorial can be shared with and reviewed by Herwig? I don't know how to preview this tutorial.

### CODE REVIEW

**Comment by florisvdh** [comment]
@hansvancalster I recommend to extend the `output` key in the Rmd YAML header as:

```
output: 
    md_document:
        preserve_yaml: true
        variant: gfm
    html_document: default
```

Knit as html (don't commit the html) and then drop the (zipped) generated html in a message here, so that you get a hyperlink.

Rendering the site with `hugo` separates css, javascript and other functionality from the actual html file, as the former are common to all pages - so it's less convenient for sharing as files. (but I do use it sometimes to check layouted results locally)

**Comment by hansvancalster** [comment]
Thanks @florisvdh I will send Herwig the html file by e-mail for review.

**Comment by hansvancalster** [comment]
@florisvdh Herwig did not have further comments. So I wait for your comments or approval to finalize this PR.

File: content/tutorials/style_inbo_bibliography/index.Rmd
Comment (florisvdh) [inline content/tutorials/style_inbo_bibliography/index.Rmd:65]:
```suggestion
There are good online tutorials that explain how to use Zotero in combination with [google docs](https://www.zotero.org/support/google_docs), [MS Word](https://www.zotero.org/support/word_processor_plugin_usage) and [LibreOffice](https://www.zotero.org/support/libreoffice_writer_plugin_usage). 
```

I saw you mentioned LibreOffice for Mendeley, so adding it here.

File: content/tutorials/style_inbo_bibliography/index.Rmd
Comment (florisvdh) [inline content/tutorials/style_inbo_bibliography/index.Rmd:32]: Once the brackets are corrected, this link will render as:

> See [finding-and-installing-styles](https://citationstyles.org/authors/#/finding-and-installing-styles) for more information.

I'm not sure if that was your intention. (It might be)

**Review by florisvdh — CHANGES_REQUESTED** [review]
Nice tutorial @hansvancalster , well illustrated! Very thoughtful that you set the language in the screenshots to English for wider adoption :+1: 

In the html output, you can see that quite some URLs are not formatted correctly. You still need to change it to `[text](url)` in several places (instead of reversing bracket types).

I've had a avant-première look at the layout, and (apart from URLs) it looks great :smile: 

In sections 'Zotero' and 'Mendeley', you best state explicitly that the approach takes the INBO CSL file _directly_ from the CSL style repository and will keep it up to date with new releases.

Please regenerate the md file accordingly after applying changes.

File: content/tutorials/style_inbo_bibliography/index.Rmd
Comment (hansvancalster) [reply content/tutorials/style_inbo_bibliography/index.Rmd:39]: Good point. Will change it.

**Comment by hansvancalster** [comment]
@florisvdh thanks for the great review (as always :+1: ). Can you have one final look? I think all concerns are addressed.

File: content/tutorials/style_inbo_bibliography/index.Rmd
Comment (florisvdh) [inline content/tutorials/style_inbo_bibliography/index.Rmd:28]:
```suggestion
EndNote uses a `*.ens` file to store the style formatting rules, whereas Mendeley and Zotero use the open standard `*.csl` ([Citation Style Language](https://citationstyles.org/)).
```
I think it's best in another pair of brackets here :wink:

File: content/tutorials/style_inbo_bibliography/index.Rmd
Comment (florisvdh) [inline content/tutorials/style_inbo_bibliography/index.Rmd:38]:
```suggestion
Curious how the formatted INBO citation style and bibliography look like for various document types?
```

File: content/tutorials/style_inbo_bibliography/index.Rmd
Comment (florisvdh) [inline content/tutorials/style_inbo_bibliography/index.Rmd:32]:
```suggestion
In case you want to manually download the latest official CSL file of the INBO citation style (not the recommended way; better use your bibliography manager): it is [available](https://github.com/citation-style-language/styles/blob/master/research-institute-for-nature-and-forest.csl) at the CSL style repository.
```

File: content/tutorials/style_inbo_bibliography/index.Rmd
Comment (florisvdh) [inline content/tutorials/style_inbo_bibliography/index.Rmd:34]:
```suggestion
To use older versions, you will need to manually download and add the file to your bibliography manager.
```

**Review by florisvdh — APPROVED** [review]
Good work @hansvancalster . Some minor suggestions below (done only in Rmd). And then ready to merge!

**Comment by florisvdh** [comment]
Olé https://inbo.github.io/tutorials/tutorials/style_inbo_bibliography/

**Pull request merged**
